rust/xaynet-server/src/state_machine/phases/unmask.rs

```diff
@@ -223,9 +223,570 @@ where
 mod tests {
     use super::*;
 
-    impl Unmask {
-        pub fn aggregation(&self) -> Option<&Aggregation> {
-            self.model_agg.as_ref()
+    use std::sync::Arc;
+
+    use anyhow::anyhow;
+    use xaynet_core::{SeedDict, SumDict};
+
+    use crate::{
+        state_machine::{
+            coordinator::CoordinatorState,
+            events::{DictionaryUpdate, EventPublisher, EventSubscriber, ModelUpdate},
+            tests::{
+                utils::{assert_event_updated, enable_logging, init_shared, EventSnapshot},
+                CoordinatorStateBuilder,
+                EventBusBuilder,
+            },
+        },
+        storage::{
+            tests::{
+                utils::{create_global_model, create_mask},
+                MockCoordinatorStore,
+                MockModelStore,
+                MockTrustAnchor,
+            },
+            Store,
+        },
+    };
+
+    fn events_from_sum_phase(state: &CoordinatorState) -> (EventPublisher, EventSubscriber) {
+        EventBusBuilder::new(state)
+            .broadcast_phase(PhaseName::Sum2)
+            .broadcast_sum_dict(DictionaryUpdate::New(Arc::new(SumDict::new())))
+            .broadcast_seed_dict(DictionaryUpdate::New(Arc::new(SeedDict::new())))
```

Comment:
i think this should be `Invalidate` since we moved that to the sum2 phase.
```suggestion
            .broadcast_sum_dict(DictionaryUpdate::Invalidate)
            .broadcast_seed_dict(DictionaryUpdate::Invalidate)
```

Reply:
good catch👍